functions/tips-scopes/main.py

```diff
@@ -28,11 +28,15 @@ def light_computation():
 
 # [START functions_tips_scopes]
```

Comment:
WDYT about moving the region tags up to include the definition of heavy_computation, and then putting the import statement above that? This would make sense in terms of being as explicit as possible, and it would pass linting.